examples/meta/src/meta_api/kwargs.sg

```diff
@@ -0,0 +1,37 @@
+# Keyword arguments are supported when constructing objects
+
+# Either when using factory functions:
+#Kernel k = kernel_machine(a=2)
+#Kernel k2 = kernel_machine("GaussianKernel", c=1.0)
+
+# Or when constructing objects from class constructors
+#GaussianKernel k3(1, a=2, b=3)
+
+# This fails (keyword arguments appear before normal arguments)
+#Kernel k4(asd=123,"SomeParam",kw=val)
+
+# This fails (keywords not allowed outside initialisations of variables)
+#k4 = kernel_machine("GaussianKernel", b=2)
+
+# This fails (keywords not allowed outside initialisation of variables)
+#Kernel k6 = kernel_machine("GaussianKernel", a=glob_fun(kw=123))
+
+# This is fine:
+#Kernel k6 = kernel_machine("GaussianKernel", a=glob_fun(ordinary_argument))
+
+# This also fails (keywords not allowed outside initialisation variables)
+#kernel_machine("GaussianKernel", a=glob_fun(kw=123))
+
+
+# REAL EXAMPLE
+# ------------
+KernelMachine svm = kernel_machine("LibSVM", C1=2.0)
+
+GaussianKernel k(log_width=2.0)
+
+GaussianKernel k2(log_width=14.0)
```

Comment:
why two kernels?

Reply:
One of them had a boolean keyword argument earlier, but that fails in python. I’ll revert it and comment it out